boards/arm/samv7/common/Kconfig

```diff
@@ -0,0 +1,73 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config BOARD_SAMV7_COMMON
+       bool "Board common logic"
+       default y
+       ---help---
+               Board common logic located in each board/common folder.
+
+if BOARD_SAMV7_COMMON
+
+config SAMV7_PROGMEM_MCUBOOT_PARTITION
+	bool
+	default n
+	select MTD
+	select MTD_BYTE_WRITE
+	select MTD_PARTITION
+	select MTD_PROGMEM
+	select MTD_PROGMEM_ERASESTATE
+
+menuconfig SAMV7_FORMAT_MCUBOOT
+	bool "MCUboot-bootable format"
+	default n
+	select ARCH_RAMFUNCS
+	select BOARDCTL_RESET
+	select BOOT_MCUBOOT
+	select SAMV7_PROGMEM
+	select SAMV7_PROGMEM_MCUBOOT_PARTITION
+	select SAMV7_SYSTEMRESET
+	---help---
+		The MCUboot support of loading the firmware images.
+
+if SAMV7_FORMAT_MCUBOOT
```

Comment:
`SAMV7_MCUBOOT_BOOTLOADER` -- config option is missing

Reply:
CONFIG_BOARDCTL_BOOT_IMAGE is selected when building the boot-loader and is enough.